### PR TITLE
Setup rails api

### PR DESCRIPTION
**Pull request by alanluqman**
## Description 
In this PR, I have created a rails API project as back-end, that serves a list of greeting text.

### Project requirements [ Back-End ]

- Create a new Rails API app called 'hello-rails-back-end'.
- Initialize your project with Git.
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table (you will need a controller with an action an rails route).
### Project requirements [ Front-End ]

- Create a new React app called 'hello-react-front-end'
- Initialize your project with Git.
- Create the `App` component with react-router.
- Create the `Greeting` component that will display a greeting. Set it up as a route in your App component.
- Create a store, an action and a reducer that will connect to you API endpont to get the random greeting.
- Display the random greeting in your `Greeting` component.

### Front-End Project
👉🏻  [Front-End](https://github.com/alanluqman/hello-react-front-end)

### CODE REVIEW

File: app/models/application_record.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
+  primary_abstract_class
```
Comment (tresorsawasawa) [inline app/models/application_record.rb:2]: - [ ] Good job with your API as it works as expected. But I noticed that there are still some things you should fix in your **[frontend](https://github.com/alanluqman/hello-react-front-end/pull/1)** pull request in order for your project to be completed. Congratulations :clap:

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @alanluqman 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow Gitflow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/application_record.rb (hunk 1 above)
Comment (alanluqman) [reply app/models/application_record.rb:2]: thanks for your review dear @tresorsawasawa , i will do.

**Review by addod19 — APPROVED** [review]
### Hi @alanluqman, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### HIGHLIGHTS
- [ ] Nice job fixing the linter errors :100: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alanluqman** [comment]
Thanks for review Dear @addod19

**Pull request merged**